### ISSUE TITLE
request.user.is_authenticated is not callable in Django 2.0

### ISSUE THREAD

**Issue by coagulant** [opbeat/opbeat_python#190]
See: https://github.com/MyBook/opbeat_python/pull/1

Sorry, don't have to to do a proper pull-request, because my fix would break exiting Django < 1.10 support or would require to vendor compatibility shims. Which is cumbersome given there is no Django tox matrix to test against.

```
TypeError: 'bool' object is not callable
  File "opbeat/handlers/logging.py", line 56, in emit
    return self._emit(record)
  File "opbeat/contrib/django/handlers.py", line 37, in _emit
    return super(OpbeatHandler, self)._emit(record, request=request)
  File "opbeat/handlers/logging.py", line 135, in _emit
    date=date, **kwargs)
  File "opbeat/contrib/django/client.py", line 165, in capture
    data.update(self.get_data_from_request(request))
  File "opbeat/contrib/django/client.py", line 153, in get_data_from_request
    result['user'] = self.get_user_info(request)
  File "opbeat/contrib/django/client.py", line 68, in get_user_info
    if request.user.is_authenticated():
```

**Comment by michael-k** [comment]
same here

**Comment by beniwohli** [comment]
Hey all! Since [Opbeat has joined Elastic](https://www.elastic.co/blog/welcome-opbeat-to-the-elastic-family), we concentrated our efforts on the all new Elastic APM (with a Beta release to drop Real Soon Now™️). Unfortunately, this means that we will not be able to get the Opbeat module to be compatible with Django 2.0.

If you want to give Elastic APM a spin, it's open source and ready for [Download](https://www.elastic.co/downloads/apm). The [Python APM agent](https://pypi.python.org/pypi/elastic-apm/) should be fully compatible with Django 2.0.

**Comment by michael-k** [comment]
Then we'll have to run our fork in production for now.

**Comment by robert-weissgraeber** [comment]
>  this means that we will not be able to get the Opbeat module to be compatible with Django 2.0.

Are you serious on this? This is a 20 line change, that you don't even have to write.

**Comment by mfa** [comment]
we will have to move (back) to sentry then.
they have 2.0 support: getsentry/raven-python#1127

**Comment by beniwohli** [comment]
Hey all

We hear your concerns, and reconsidered the decision on Django 2.0. I just opened a PR that should fix this and other issues (mostly with our test suite): #193. It's slightly more than 20 lines ;) 

I will create a new release on PyPI once the tests pass, this will probably take a couple hours

**Comment by beniwohli** [comment]
I just pushed version 3.6 to PyPI, which contains the fixes from #193.
